tests/unit/records/test_records_receivers.py

```diff
@@ -705,9 +564,8 @@ def test_populate_inspire_document_type_doc_type_from_valid_primary_and_lectures
 
 def test_populate_inspire_subjects_preserves_terms_from_inspire():
```

Comment:
`def test_populate_inspire_subjects_preserves_terms` (the `scheme` is no longer relevant).